docs/README.md

```diff
@@ -1696,28 +1698,23 @@ Knowing this, you can infer the relationship between those operations and your B
 - You _subscribe_ to a **read model** using a **subscription** 
 
 #### How to send GraphQL request
-GraphQL uses two existing protocols: 
-
-- _HTTP_ for `mutation` and `query` operations
-- _WebSocket_ for `subscription` operations
-
-The reason for the WebSocket protocol is that, in order for subscriptions to work, there must be a way for the server to send data
-to clients when it is changed. HTTP doesn't allow that, as it is the client the one which always initiates the request.
+GraphQL uses two existing protocols _HTTP_ and _WebSocket_. The reason for the WebSocket protocol is that, in order for subscriptions to work, there must be a way for the server to send data
```

Comment:
The reason for the WebSocket protocol is to allow subscriptions to work. There must be a way for the server to send data
to clients when there is a change in a subscription. HTTP doesn't allow that, as it is always the client the one who initiates the request.

Reply:
@rdiaz82 Did you mean to change the text to what you wrote? If so, which part did you changed?